app/controllers/concerns/authentication_and_sso_concerns.rb

```diff
@@ -105,6 +105,16 @@ def set_session_expiration_header
     headers['X-Session-Expiration'] = @session_object.ttl_in_time.httpdate if @session_object.present?
   end
 
+  def log_sso_info
+    action = request.controller_instance.controller_path.classify.to_s
+    action += 'Controller#'
+    action += request.parameters['action'].to_s
+
+    Rails.logger.warn(
```

Comment:
Should this be `Rails.logger.info`?

Reply:
Yes.  good catch.  I'll update.

Reply:
historically we set all direct deposit logs to warn just in case we needed to turn off "info logs". This has only happened once in the past where the logging solutions was caused issues with vets-api. So I suspect the same model is applied to other direct deposit related logs. I dont think we need to continue to do warn for all of these but wanted to leave that door open.

Reply:
@joeniquette Should this be applied to `GET` requests as well?  Right now, it's only on update requests.